elements/rh-tabs/docs/10-style.md

```diff
@@ -0,0 +1,160 @@
+Tabs can be used in light and dark themes and there are two variants to choose 
+from. Open tabs feature a more understated or cleaner visual style whereas Box 
+tabs feature a familiar box or container visual style.
+
+{% example palette="light",
+           width=872,
+           class="centered",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Examples of tabs style",
+           src="../tabs-style.svg" %}
+
+## Theme
+
+{% example palette="light",
+           width=872,
+           class="centered",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Light theme tabs",
+           src="../tabs-theme-light.svg" %}
+
+{% example palette="darkest",
+           width=872,
+           class="centered",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Dark theme tabs",
+           src="../tabs-theme-dark.svg" %}
+
+{% example palette="light",
+           width=872,
+           class="centered",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Light theme box tabs",
+           src="../tabs-box-theme-dark.svg" %}
+
+{% example palette="darkest",
+           width=872,
+           class="centered",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Dark theme box tabs",
+           src="../tabs-box-theme-dark.svg" %}
+
+## Orientation
+
+Both Tab variants have horizontal and vertical orientations. The styles are the 
+same between orientations, only the arrangement of elements is different.
+
+Open tabs feature section text labels that float above the anchor line in the 
+horizontal orientation or to the right of the anchor line in the vertical 
+orientation.
+
+{% example palette="lightest",
+           width=872,
+           class="inline-stacked",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Open tabs orientation",
+           src="../tabs-orientation-center.svg" %}
+{% example palette="lightest",
+           width=708,
+           class="inline-stacked",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Open tabs orientation",
+           src="../tabs-orientation-overflow.svg" %}
+{% example palette="lightest",
+           width=85,
+           class="inline-stacked",
+           style="margin-block:var(--rh-space-2xl);",
+           alt="Open tabs orientation",
+           src="../tabs-orientation-vertical.svg" %}
```

Comment:
```Suggestion
{% example palette="lightest",
           width=240,
           class="inline-stacked",
           style="margin-block:var(--rh-space-2xl); width: 240px;",
           alt="Open tabs orientation",
           src="../tabs-orientation-vertical.svg" %}
```

The image is super large, being a vertical tab screenshot and image is filling 100% width.  Setting 240 in the width attribute in the example shortcode didn't seem to work, adding the same width to the style attribute did work.  Assuming maybe the shortcode has a bug with the width attribute?

Reply:
instead, the fix appears to be wrapping these examples in a {% section %}...{% endsection %} which provides the `.section` class needed for the styles to work correctly